plugins/fabric8-multi-tenant-manager/pom.xml

```diff
@@ -86,6 +116,11 @@
             <groupId>org.eclipse.che.core</groupId>
             <artifactId>che-core-commons-inject</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.che.core</groupId>
+            <artifactId>che-core-commons-lang</artifactId>
+            <version>6.7.1</version>
```

Comment:
@amisevsk am I correct that in final PR it is not planned to hardcode the dependency versions, but rather use the one provided from parent ?

Reply:
Yeah sorry, I quickly added the dependencies to fix the enforce phase of the build to get comments sooner. I'll fix this.

Reply:
Fixed.